Woof/Woof/ViewModel/StartScreenViewModel.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// Responsible for managing the data and behavior of the start screen view.
+final class StartScreenViewModel: ObservableObject {
+    /// The user role that defines the app flow, it's `none` by default.
+    @Published var userRole: Role = .none
```

Comment:
And why assign the property the role of .none, if .none is assigned during initialization, if the role is not selected?

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/156/commits/2291d80754fc4ddd7c76e05f5b0e21f133902573